From 8f07768085fa4c8d21dbe27805b7238e67d3aea5 Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Tue, 17 May 2022 15:52:58 -0600
Subject: [PATCH] Move pytest-runner to tests_require

This isn't a build requirement so it shouldn't be under
setup_requires.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[Upstream status:
https://github.com/cenobites/flask-jsonrpc/pull/227]
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 197d6c9..f23c472 100755
--- a/setup.py
+++ b/setup.py
@@ -42,8 +42,8 @@ setuptools.setup(
         'async': ['Flask[async]>=1.0.0,<3.0'],
         'dotenv': ['Flask[dotenv]>=1.0.0,<3.0'],
     },
-    setup_requires=['pytest-runner'],
     tests_require=[
+        'pytest-runner',
         'mock==4.0.3',
         'coverage==6.3.2;python_version>"3.6"',
         'coverage<6.2;python_version<="3.6"',
-- 
2.36.0

